random.cabal

```diff
@@ -108,7 +107,7 @@ test-suite legacy-test
     default-language: Haskell2010
     ghc-options:      -with-rtsopts=-M4M -Wno-deprecations
     build-depends:
-        base >=4.10 && <5,
+        base -any,
```

Comment:
Any particular reason for removing the version bounds here and further down?

Reply:
There is no need to update the same bounds in multiple places. All: tests and benchmarks depend on `random`, which decides bounds for `base` and other packages the library depends on. So this duplication can only cause problems. Consider in the future we need to adjust upper bound on base, you must not forget to do it for all sections, which is redundant and error prone. In fact I already fell onto this when I was updating splitmix bounds